plugins/inputs/f5_load_balancer/README.md

```diff
@@ -0,0 +1,72 @@
+# F5 Load Balancer Input Plugin
```

Comment:
What version of the REST API does this plugin support? could you mention it in the README. Perhaps also including a link to the REST API documentation would be heplful for people wanting to use this plugin.